libmscore/sym.cpp

```diff
@@ -5321,6 +5321,7 @@ QVector<oldName> oldNames = {
       {"32' rest",                              SymId::rest32nd },            // rests.5
       {"64' rest",                              SymId::rest64th },            // rests.6
       {"128' rest",                             SymId::rest128th },           // rests.7
+      {"256' rest",                             SymId::rest256th },           // rests.8
```

Comment:
I'll put it in and also do the sync with `fonts/mscore/glyphnames.json` (totally missed this one!) and if we decide not to put it in with 2.0.4 then it can be cut. But I think if we do end up using this PR it would be best to aim for 2.0.4/5 since the difference in tick division will make it a huge pain to keep tests synchronized between branches otherwise.